src/pudl/metadata/enums.py

```diff
@@ -215,3 +266,45 @@
     "UtilityPV",
 }
 """NREL ATB technology descriptions."""
+
+
+TECH_DESCRIPTIONS_EIAAEO: list[str] = [
+    "coal",
+    "combined_cycle",
+    "combustion_turbine_diesel",
+    "distributed_generation",
+    "diurnal_storage",
+    "fuel_cells",
+    "natural_gas",
```

Comment:
Added `natural_gas`, `petroleum`, and `pumped_storage_other` which show up in the "generation by fuel type". 

I think the dimension for the "generation_by_fuel_type" is still a "tech description" and not a "fuel type", though:

```
    Coal
    Petroleum
    Natural Gas
    Nuclear
    Pumped Storage and Other 13/
    Renewable Sources 14/
    Distributed Generation 6/
 ```
So I'm ok shoving those into the same bucket.